crates/daemon/Cargo.toml

```diff
@@ -39,3 +39,10 @@ ed25519-dalek.workspace = true
 pem.workspace = true
 rand.workspace = true
 rusty_paseto.workspace = true
+
+# Testing
+[dev-dependencies]
+assert_cmd = "2.0.14"
```

Comment:
We follow a convention of having dependency versions declared in the root workspace Cargo.toml, and then use .workspace = true style dependencies in the workspace crates. This way we're attempting to enforce one version of deps for all crates without risk of drift. Even for dev deps.

Reply:
Ah right. Will do.